content/blog/2023-10-05-october-Prism-launch/contents.lr

```diff
@@ -0,0 +1,22 @@
+title: Prism: The New Feinberg Repository for Global Dissemination of Research
+---
+pub_date: 2023-10-10
+---
+team: InvenioRDM
+---
+body:
+
+![](NEW_PICTURE_FILE_NEEDED.jpg)
+
+Feinberg School of Medicine has a newly available institutional repository called Prism. Prism preserves the intellectual works created by the Feinberg scholarly community and holds articles, conference presentations, preprints and other items created by faculty, staff, and students. Prism helps openness, maximizes reproducibility, and enhances research connections in the Feinberg community and globally to anyone with access to the internet. Prism will be available to Feinberg in April 2023.
```

Comment:
We probably want to cut

> Prism will be available to Feinberg in April 2023.

out.

Reply:
Since this is an announcement after the fact, maybe we could say: "Prism was made available to the Feinberg community in April 2023."